pointgrey_camera_driver/CMakeLists.txt

```diff
@@ -100,6 +101,5 @@ if (CATKIN_ENABLE_TESTING)
   roslaunch_add_file_check(launch/camera.launch)
   roslaunch_add_file_check(launch/stereo.launch)
 
-  find_package(roslint REQUIRED)
```

Comment:
The issue is that `roslint` is only a `test_depend`, so it needs to only be looked for inside the `if(CATKIN_ENABLE_TESTING)` block.